improver_tests/acceptance/SHA256SUMS

```diff
@@ -315,13 +315,13 @@ b60f6046c86319f8b7ca3b5d7902dbaf3a52f571f30ba56a1a4bc814c42dd341  ./combine/mini
 cd5fe4e4ef61d890c30cc9cbd236bf0dfdbedd5f12f8a92803aa57be84c0d9ab  ./combine/multiplication_cellmethods/kgo.nc
 f1ae76b9374c5d1076b89a7348fe9bbc393a12ae4ccdc660a170ba5ff0f823ab  ./combine/multiplication_cellmethods/precipitation_accumulation-PT01H.nc
 b8934494b4a24daa2408c4d95a2367e328e25e8323e34c67ef6026d51021be32  ./combine/multiplication_cellmethods/precipitation_is_snow.nc
-0bd96af6cb5c6caa045e397589dd0ce3b498af837d989fe73326f5e9459c6054  ./construct-reliability-tables/basic/forecast_0.nc
-fbc14286b4ce41e2e60df0870ae4911c1b00a38ec96912f43c6187fcaf7d02f6  ./construct-reliability-tables/basic/forecast_1.nc
+0bd96af6cb5c6caa045e397589dd0ce3b498af837d989fe73326f5e9459c6054  ./construct-reliability-tables/basic/forecast/forecast_0.nc
+fbc14286b4ce41e2e60df0870ae4911c1b00a38ec96912f43c6187fcaf7d02f6  ./construct-reliability-tables/basic/forecast/forecast_1.nc
 0d0edf9751a2019db952907700b02499ec9f1c360db4591a8012ca247a841c73  ./construct-reliability-tables/basic/kgo_aggregated.nc
 902e5cb9d3dc5d2b78bb99aff8370f9815adf5064b2caeb7abed73a56a897a43  ./construct-reliability-tables/basic/kgo_single_value_bins.nc
 72d4fd0655d1b7a2bc11d85741ec944f195c59813ae629e6858116c4e09eccb0  ./construct-reliability-tables/basic/kgo_without_single_value_bins.nc
-8ed50464c34b8673d98d1256d1c11b9eeea911dc79f7f75d425a590bf8697301  ./construct-reliability-tables/basic/truth_0.nc
-3999adb3749052d9efdfab863427a20a1fabbca06ff430c6c9cf5f89d1ea4d60  ./construct-reliability-tables/basic/truth_1.nc
+8ed50464c34b8673d98d1256d1c11b9eeea911dc79f7f75d425a590bf8697301  ./construct-reliability-tables/basic/truth/truth_0.nc
+3999adb3749052d9efdfab863427a20a1fabbca06ff430c6c9cf5f89d1ea4d60  ./construct-reliability-tables/basic/truth/truth_1.nc
```

Comment:
I've found best practices around the acceptances tests to be fairly obscure. This change is required to match the updated expected inputs for the cli, but it is unclear to me how we would implement this change for all users? And how much of a pain it would be?

If this is too much of a pain, I could make some changes to `get_cube_from_directory`